### PR TITLE
update vi-first-steps.md

### LINKED ISSUE #2507: Adding checklists at the end of each step
### Problem
In each of the steps, the virtual interns are required to complete a certain set of tasks. However, as of right now, there isn't really a checklist where they could check if they did all of the required tasks for each of the steps. 

### Steps to reproduce the problem

N/A

### Screenshots

Screen Shot 2019-07-01 at 10 39 54 AM

### Proposed solution

Similar to the text given in Step 3, I think it would be helpful for interns to have a checklist for each step. Where the text could say something along the lines of: 
"Once you complete Step #, you will have (1) (2) and (3)"

### PR DESCRIPTION
**Pull request by hemu-codes**
Fixes #2507 

### Description
This is a pull request for issue #2507 and I updated to the original page by adding a summary of what is done in each step. This will help users in getting back to the right step if they forget how to do a certain task. 

### Raw.Githack preview link

https://raw.githack.com/hemu-codes/hemu-codes.github.io/add-step-summaries/#!pages/vi/vi-first-steps.md

### CODE REVIEW

File: pages/vi/vi-first-steps.md [hunk 1]
```diff
@@ -49,11 +49,17 @@ There are 3 sections in this step:
 
 1. [Vagrant Tutorial](vi-vagrant.md)
   **Vagrant** is an open-source software product for building and maintaining portable virtual software development environments. In this section, you will learn about how to interact with Vagrant through the command-line interface.
+  
+* **Summary of Step 1:**
```
Comment (Liew211) [inline pages/vi/vi-first-steps.md]: The bullet points and the "Summary of Step X" are not necessary.

File: pages/vi/vi-first-steps.md [hunk 2]
```diff
@@ -49,11 +49,17 @@ There are 3 sections in this step:
...
+    * By the end of this step, you should know how to set up your planet community on the command line/terminal, creating your planet learning account and learning about the different commands that will be used on the terminal for working with vagrant.
```
Comment (Liew211) [inline pages/vi/vi-first-steps.md]: This part is not necessary; it is already stated in the previous paragraph.

File: pages/vi/vi-first-steps.md [hunk 3]
```diff
@@ -86,6 +98,9 @@ Make sure you have created at least one issue, resolved it, commented on an issu
 
 Follow the directions under the [Nation Planet tab](vi-nation.md).
 
+* **Summary of Step 7:**
+    * In this step you learn about syncing your planet community with nation and also more on how to manage and checking the status of your syncs.
```
Comment (Liew211) [inline pages/vi/vi-first-steps.md]: Grammar - checking -> check

**Comment by lmmrssa** [comment]
@hemu-codes I don't see need of adding any of these summary. Because first paragraph of each step is explaining what step is about. And you will be taken to another detail page to do more stuff in there. So basically this overview page is itself a summary page.

**Comment by hemu-codes** [comment]
@lmmrssa I thought of adding this part due to the fact that the first paragraph of each step seems to have a lot of details that you might not necessarily want to read through if all you are looking for is where you learned how to do a certain step of the process.

**Comment by Liew211** [comment]
@lmmrssa I think some of this information could be useful though, when looking back at the steps i.e. to differentiate between "Planet tutorial" and "Nation Planet" when looking for the description of a specific feature.

**Comment by hemu-codes** [comment]
@Liew211 I have made the changes requested and notified you on Gitter but I wasn't sure if you had the chance to look at it again. Could you please look over this? Thanks

**Comment by lmmrssa** [comment]
@Liew211 @hemu-codes we do not want people to just rush and jump (skip) details. Otherwise we can just remove all details and make the 10 steps short. We want you to go through all details and learn the idea not just run some command and get it done.

**Comment by hemu-codes** [comment]
@lmmrssa The summary is not meant to give away everything we did in the whole step but rather just a refresher to what they did so if they need to go back to where they learned how to do a certain task, they don't have to browse through each page to know where to go.

**Comment by ChiragAwale** [comment]
@hemu-codes I suggest you work on a different issue. Also, it does not need to be a big change. Small crucial/helpful changes are all welcome.

**Pull request closed without merging**